packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -888,7 +888,7 @@ class CliServiceProvider extends ServiceProviderCore implements ServiceProvider
    * @returns {Promise} topology.
    */
   getTopology(): Topology | undefined {
-    return this.mongoClient.topology;
+    return this.mongoClient.topology; // TODO: maybe return topology description instead, see NODE-2910
```

Comment:
I think I’m missing something re: this comment/the corresponding one in `service-provider-core` – Right now, we can do `serviceProvider.getTopology()?.description` to access the description, and that’s publicly accessible without hitting any private properties. What do we need from the driver here?

Reply:
From our conversation last week they said that topology description was something they'd consider making public directly in the future, https://jira.mongodb.org/browse/NODE-2910, but if that's gone away bceause it's already public we can should the ticket.

Reply:
I think the difference between `getTopologyDescription` and just `topology.description` is minor enough we should just close the ticket, save everyone some time

Reply:
Yeah, I’ve closed the ticket. Maybe they were referring to making only the topology description available, rather than the public `Toplogy` class + topology description? But I think in that case we should also be fine.

Reply:
Yeah, either way the comment should be gone :)